__tests__/tests.js

```diff
@@ -50,7 +50,9 @@ describe('htmlbars-inline-precompile', function () {
     );
 
     expect(transpiled).toMatchInlineSnapshot(`
-      "var compiled = Ember.HTMLBars.template(
+      "import { createTemplateFactory as _createTemplateFactory } from \\"@ember/template-factory\\";
```

Comment:
Why does it alias?

Reply:
it aliases due to using the `path.scope.generateUidIdentifier`  function, which as far as I can tell always does this to create a unique identifier

Reply:
k, gotcha